tests/test_bundle_side_input.py

```
@@ -0,0 +1,391 @@
+# ---------------------------------------------------------------------------
+# compare_release_against_bundle_facts -- real compiled binaries
+# ---------------------------------------------------------------------------
+
+
+@pytest.mark.integration
+class TestCompareReleaseAgainstBundleFacts:
+    def _build_so(self, tmp_path: Path, name: str, body: str) -> Path:
+        src = tmp_path / f"{name}.c"
+        src.write_text(body)
+        out = tmp_path / name
+        res = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)